lib/global-admin/addon/components/cru-cloud-credential/component.js

```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
   },
 
   setError(message = '') {
```

Comment:
it looks like this method is called from an OCI auth validation failure with an error message. Is it empty?  Is it possibly more descriptive then the standard message that is in the translation bundle?  Was it too descriptive and deemed inappropriate to display?

Reply:
It was too descriptive and we decided the general one was better.

Reply:
It's not empty. It is pretty descriptive but we can show it. 
The error for the specific case you filed is 
"can not create client, bad configuration: PEM data was not found in buffer: There was a problem validating your keys. Enter valid credentials and try again."